src/App.tsx

```diff
@@ -15,48 +16,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export const rev = 'reverse';
+export const alphabet = 'alphabet';
+export const alphabetRev = 'alphabet reverse';
+export const length = 'length';
+export const lengthRev = 'length reverse';
+
 export const App: React.FC = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [sortField, setSortField] = useState('');
+
+  const searchClass = (allClass: string) => {
+    if (allClass.includes(rev)) {
+      return true;
+    }
+
+    return false;
+  };
+
+  const classButton = (buttonName: string, classButtonActive: string) => {
```

Comment:
Same here
![image](https://github.com/mate-academy/react_list-of-goods-hooks/assets/96409563/f91d83ee-ed70-4e1d-9aac-1d79da7fb00f)